### PR TITLE
Add backend folder structure, add eslint, prettier, nodemon and seque…

### LINKED ISSUE #7: Backend setup
- [x] Setup folder structure
- [x] Add Sequelize migrations
- [x] Add Eslint config
- [x] Add logger setup

### PR DESCRIPTION
**Pull request by Pr1sm0**
…lize config, add sequelize with migrations

### CODE REVIEW

File: server/src/app.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import * as express from 'express';
+import * as cors from 'cors';
+import * as winston from 'winston';
+import * as dotenv from 'dotenv';
+import routes from './routes/index';
+
+dotenv.config();
+
+const PRODUCTION = 'production';
```
Comment (andrii-kuzmichov) [inline server/src/app.ts]: Move to constants file

File: server/src/app.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+const PORT: number = Number(process.env.PORT) || 8080;
```
Comment (andrii-kuzmichov) [inline server/src/app.ts]: The same as above

File: server/src/app.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+const app = express();
+
+app.use(cors());
+app.use(express.json({ limit: '100mb', type: 'application/json' }));
```
Comment (andrii-kuzmichov) [inline server/src/app.ts]: Create constants for 100mb and type

File: server/src/app.ts [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+app.use(express.urlencoded({ extended: true }));
+app.use('/api/v1/', routes);
```
Comment (andrii-kuzmichov) [inline server/src/app.ts]: create constants for path

File: server/src/app.ts [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+export const logger = winston.createLogger({
+  level: 'info',
+  format: winston.format.json(),
+  defaultMeta: { service: 'user-service' },
+  transports: [
+    //
+    // - Write all logs with level `error` and below to `error.log`
+    // - Write all logs with level `info` and below to `combined.log`
+    //
```
Comment (andrii-kuzmichov) [inline server/src/app.ts]: clean up comments

File: server/src/db/sequelize/models/index.ts [hunk 6]
```diff
@@ -0,0 +1,16 @@
+import { Sequelize } from 'sequelize';
+
+const env = process.env.NODE_ENV || 'development';
```
Comment (andrii-kuzmichov) [inline server/src/db/sequelize/models/index.ts]: create constant for `development`

File: server/src/routes/index.ts [hunk 7]
```diff
@@ -0,0 +1,12 @@
+import * as express from 'express';
+import { Request, Response } from 'express';
+
+const router = express.Router();
+
+router.get('/', (req: Request, res: Response) => {
+  res.status(200).send({
```
Comment (andrii-kuzmichov) [inline server/src/routes/index.ts]: create method for handle response

Comment (andrii-kuzmichov) [reply server/src/routes/index.ts]: And add constant for 200 status and other

File: server/src/app.ts [hunk 8]
```diff
@@ -0,0 +1,47 @@
+import * as express from 'express';
+import * as cors from 'cors';
+import * as winston from 'winston';
+import * as dotenv from 'dotenv';
+import routes from './routes/index';
+import * as appConstants from './constants/app';
```
Comment (andrii-kuzmichov) [inline server/src/app.ts]: Probably would be better `import {JSON_LIMIT, etc} from 'constants/app'`

**Review by babchukbohdan — COMMENTED** [review]
All good.

**Review by andrii-kuzmichov — APPROVED** [review]

**Review by babchukbohdan — APPROVED** [review]

**Review by romanh24 — APPROVED** [review]

**Pull request merged**